backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/invoicecandidate/api/impl/AggregationEngine.java

```diff
@@ -376,6 +376,7 @@ private void addToInvoiceHeader(
 			invoiceHeader.setC_BPartner_SalesRep_ID(icRecord.getC_BPartner_SalesRep_ID());
 			invoiceHeader.setC_Order_ID(icRecord.getC_Order_ID());
 			invoiceHeader.setPOReference(icRecord.getPOReference()); // task 07978
+			invoiceHeader.setEmail(icRecord.getEMail());
```

Comment:
please add "Email" to the `invoicing-agg-std`,  invoice-aggregation rule

Reply:
Done